pkg/azure/aks/price_store.go

```diff
@@ -190,15 +186,16 @@ func (p *PriceStore) validateMachinePriceIsRelevantFromSku(ctx context.Context,
 	return true
 }
 
-func (p *PriceStore) PopulatePriceStore(ctx context.Context) error {
+func (p *PriceStore) PopulatePriceStore() {
 	startTime := time.Now()
-	p.logger.Info("populating price store")
+	ctx := context.TODO()
```

Comment:
Same nit as above, this context should still be passed in.

Reply:
yup, good idea - done!